JavaScript/JavaScriptSDK.Tests/CheckinTests/Sender.tests.ts

```diff
@@ -759,7 +827,7 @@ class SenderTests extends TestClass {
                 // setup
                 XMLHttpRequest = <any>(() => {
                     var xhr = new this.xhr;
-                    xhr.withCredentials = false;
+                    delete xhr.withCredentials;
```

Comment:
This is to pretend that the browser is IE - see [Sender](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK/Sender.ts#L148)